### PR TITLE
updating css variables

### PR DESCRIPTION
**Pull request by iruzevic**
# Description

- updating css variables and inline logic to use our internal store for storing data.

### CODE REVIEW

File: scripts/editor/output-css-variables.js [hunk 1]
```diff
@@ -401,6 +278,221 @@ export const getUnique = () => {
...
+export const getDifference = (array1, array2) => {
+	return array1.filter(object1 => {
+		return !array2.some(object2 => {
+			return object1.clientId === object2.clientId;
+		});
+	});
+};
+
+/**
+ * Return css variables in default type. On the place where it was called.
```
Comment (goranalkovic-infinum) [inline scripts/editor/output-css-variables.js:365]:
```suggestion
 * Return CSS variables in default type. On the place where it was called.
```

File: scripts/editor/output-css-variables.js [hunk 2]
```diff
@@ -401,6 +278,221 @@ export const getUnique = () => {
...
 
+// ------------------------------------------------------------------------------
+// PRIVATE METHODS
+// ------------------------------------------------------------------------------
+
+// Internal variable for storing caches breakpoint data.
+let breakpointsPreparedVariableDataCache = [];
+
+/**
+ * Output css variables as a one inline style tag.
```
Comment (goranalkovic-infinum) [inline scripts/editor/output-css-variables.js:289]:
```suggestion
 * Output CSS variables as a one inline style tag.
```

**Review by gabriel-glo — APPROVED** [review]

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]
2 very minor comments, awesome job!

**Review by simun-ivanac — DISMISSED** [review]

**Review by dadadavorin — DISMISSED** [review]
Nice 👍

**Review by goranalkovic-infinum — DISMISSED** [review]

**Review by iobrado — DISMISSED** [review]

**Review by mbmjertan — DISMISSED** [review]

**Review by dingo-d — DISMISSED** [review]

**Review by volfkarlo — DISMISSED** [review]
👏  Application-like things! Awesome! 😄

**Review by dadadavorin — DISMISSED** [review]

**Review by iobrado — APPROVED** [review]

File: scripts/editor/options.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import { getPaletteColors } from "./get-palette-colors";
-import { getAttrKey } from './../helpers/check-attr';
 import _ from 'lodash';
+import { getPaletteColors } from "./colors";
```
Comment (goranalkovic-infinum) [inline scripts/editor/options.js:2]:
```suggestion
import { getPaletteColors } from './colors';
```
for consistency 😄

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]
1 more ultra-minor thing 😄

**Review by iobrado — APPROVED** [review]

**Review by dadadavorin — APPROVED** [review]

**Review by dingo-d — DISMISSED** [review]

**Review by volfkarlo — APPROVED** [review]
Awesome stuff!

**Pull request merged**